src/loader.nit

```diff
@@ -966,6 +936,15 @@ class ModulePath
 	redef fun to_s do return filepath
 end
 
+redef class MProject
+	# The associated `.ini` file, if any
+	#
+	# The `ini` file is presented and might contains invalid or missing information.
```

Comment:
Reading this a second time, I don't understand the `is presented`. What are you trying to say here?